### DISCUSSION TITLE
小米官方发布了ha集成，之后会考虑兼容官方吗

### DISCUSSION THREAD

**Discussion by FreeFlying0** [al-one/hass-xiaomi-miot#2090] [General]
我安装一下官方的，发现其控制选项会更多一些

**Comment by iKira** [comment]
使用体验上还是有点区别，官方插件添加出来的小爱音箱实体，没有一体化播放控件（包含播放、暂停、音量、上一首、下一首），每个操作都要单独添加实体。
